verifact/forum/schema.py

```diff
@@ -23,27 +23,36 @@ class Meta:
         interfaces = (relay.Node,)
         convert_choices_to_enum = False
 
+class VoteNode(DjangoObjectType):
+    class Meta:
+        model = Vote
+        interfaces = (relay.Node,)
 
 class QuestionConnection(relay.Connection):
     class Meta:
         node = QuestionNode
 
-
 class AnswerConnection(relay.Connection):
     class Meta:
         node = AnswerNode
 
+class VoteConnection(relay.Connection):
+    class Meta:
+        node = VoteNode
 
 class Query(ObjectType):
     questions = relay.ConnectionField(QuestionConnection)
     answers = relay.ConnectionField(AnswerConnection)
+    votes = relay.ConnectionField(VoteConnection)
 
     def resolve_questions(root, info):
         return Question.objects.all()
 
     def resolve_answers(root, info):
         return Answer.objects.all()
 
+    def resolve_votes(root, info):
```

Comment:
I don't think we need this on the root Query object. It will be available through the `AnswerNode` automatically. I actually think we can remove the `answers` and `votes` from the `query` object here without any adverse effects. The `questions` we still need because questions are public and won't be available through the `viewer` object. But everything related to questions should be available through that graph. Does that make sense?

Reply:
Yea I was thinking about this recently because I thinking of implementing a `count_credible_by_answer_id` within the AnswerNode.

Does the second-to-last sentence mean that I can technically access a `Vote` via it's corresponding `answer_id` which is in turn accessed via it's corresponding `question_id`?

Reply:
Yup, that's right. You can access a `VoteNode` through an `AnswerNode` which can be accessed through the `Node` interface, that's what the `node` is in the root schema. As long as those objects are public, they can all be accessed through their respective graph relationships.